### PR TITLE
Localize numbers dates values

### PR DESCRIPTION
**Pull request by stiebitzhofer**

### CODE REVIEW

File: template.hbs [hunk 1]
```diff
@@ -229,8 +229,7 @@
       {{#each Certificate.CommercialTransaction.SupplementaryInformation}}
       <tr class="table-rows">
         <th class="table-headers" colspan="2">{{@key}} {{this.Key}}</th>
-        <td class="table-cells">{{ localizeValue this.Value this.Type ../Certificate.CertificateLanguages.[0] }}
-          {{this.Unit}}</td>
+        <td class="table-cells">{{localizeValue this.Value ../../Certificate.CertificateLanguages.[0]}} {{this.Unit}}</td>
```
Comment (getlarge) [inline template.hbs:232]: For all places where `localizeValue` helper is used we need `this.Type` to be passed too

File: template.hbs [hunk 2]
```diff
@@ -480,7 +473,7 @@
     {{#if this}}
     <table class="main-table">
       <tr class="table-rows">
-        <th class="table-headers">{{@key}} {{ i18n @key "certificateFields" ../Certificate.CertificateLanguages }}
+        <th class="table-headers">{{@key}} {{ i18n @key "certificateFields" ../Certificate.CertificateLanguages }}LLLLL
```
Comment (getlarge) [inline template.hbs:476]: `LLLLL` typo ?

**Review by getlarge — CHANGES_REQUESTED** [review]

File: template.hbs (hunk 2 above)
Comment (stiebitzhofer) [reply template.hbs:476]: Typo (or marker from testing :-()

File: template.hbs [hunk 3]
```diff
@@ -229,8 +229,7 @@
       {{#each Certificate.CommercialTransaction.SupplementaryInformation}}
       <tr class="table-rows">
         <th class="table-headers" colspan="2">{{@key}} {{this.Key}}</th>
-        <td class="table-cells">{{ localizeValue this.Value this.Type ../Certificate.CertificateLanguages.[0] }}
-          {{this.Unit}}</td>
+        <td class="table-cells">{{localizeValue this.Value this.Type ../../Certificate.CertificateLanguages.[0]}} {{this.Unit}}</td>
```
Comment (getlarge) [inline template.hbs:232]: Not sure the path of `Certificate.CertificateLanguages` is correct here.

File: template.hbs [hunk 4]
```diff
@@ -583,8 +576,7 @@
         {{#each Certificate.Validation.SupplementaryInformation}}
         <tr class="table-rows">
           <th class="table-headers" colspan="2">{{@key}} {{this.Key}}</th>
-          <td class="table-cells">{{ localizeValue this.Value this.Type ../Certificate.CertificateLanguages.[0] }}
-            {{this.Unit}}</td>
+          <td class="table-cells">{{localizeValue this.Value this.Type ../../Certificate.CertificateLanguages.[0]}} {{this.Unit}}</td>
```
Comment (getlarge) [inline template.hbs:579]: Same as above

**Pull request merged**